compat/media/camera_service.cpp

```diff
@@ -48,9 +46,7 @@ int main(int argc, char** argv)
     // for creating a new IMediaRecorder (MediaRecorder) instance over Binder
     MediaRecorderFactory::instantiate();
     // Enable audio recording for camera recording
-#if ANDROID_VERSION_MAJOR==5 && WANT_UBUNTU_CAMERA_HEADERS
     CameraRecordService::instantiate();
-#endif
     CameraService::instantiate();
```

Comment:
Have you checked it does not conflict with minimediaservice? I left only CameraRecordService running on Halium 9 (see https://github.com/Halium/libhybris/commit/a76968e17894836e93b90340409e78048a70ec32).

Reply:
You mean CameraService conflicting with minimediaservice? I have both processes running without issues but will happily test your additional `#ifdef` on 7.1.

Reply:
Done, doesn't seem to be necessary.